packages/cysync-core/src/store/accountSync/index.ts

```diff
@@ -28,6 +28,9 @@ export const accountSyncSlice = createSlice({
     setAccountSyncState: (state, payload: PayloadAction<AccountSyncState>) => {
       state.syncState = payload.payload;
     },
+    setSyncError: (state, payload: PayloadAction<string>) => {
+      state.syncError = payload.payload;
```

Comment:
- Also set `syncState` to `failed` when setting `syncError`
- When are we resetting `syncError` once it has appeared?

Reply:
changed